guides/leon/debain-email.md

```diff
@@ -0,0 +1,308 @@
+# Debian Email Box
+
+## Summary 
+This is the setup guide for the Debian Email box. We will be using a Postfix, dovecot, and webmail setup.

```

Comment:
Try to explain more about the service. Maybe have a table with the ports and visual image of the email route.

![image](https://user-images.githubusercontent.com/26574828/35924786-087096b8-0bf2-11e8-90c6-6cfb36e162fe.png)

Reply:
Also follow the template, the entire thing will flow with everyone else guide. Summary should be above after Prerequisites.